_monitoring-your-cluster/pa/index.md

```diff
@@ -8,57 +8,106 @@ redirect_from:
   - /monitoring-plugins/pa/index/
 ---
 
-# Performance analyzer
+# Performance Analyzer
 
-Performance analyzer is an agent and REST API that allows you to query numerous performance metrics for your cluster, including aggregations of those metrics. 
+Performance Analyzer is an agent and REST API that allows you to query numerous performance metrics for your cluster, including aggregations of those metrics. 
 
-The performance analyzer plugin is installed by default in OpenSearch version 2.0 and higher.
+The Performance Analyzer plugin is installed by default in OpenSearch version 2.0 and higher. If you want to use OpenSearch 2.0 or later with Performance Analyzer disabled, see [Disable Performance Analyzer](#disable-performance-analyzer).
 {: .note }
 
-## Performance analyzer installation and configuration
+## Prerequisites
 
-The following sections provide the steps for installing and configuring the performance analyzer plugin.
+Review the following prerequisites before installing Performance Analyzer.
 
-### Install performance analyzer
+### Storage
+
+Performance Analyzer uses `/dev/shm` for temporary storage. During heavy workloads on a cluster, Performance Analyzer can use up to 1 GB of space.
+
+Docker, however, has a default `/dev/shm` size of 64 MB. To change this value, you can use the `docker run --shm-size 1gb` flag or [a similar setting in Docker Compose](https://docs.docker.com/compose/compose-file#shm_size).
```

Comment:
Is it recommended to increase the value?

Reply:
Yes, otherwise, OpenSearch might not run with PA at all. We recommend increasing the memory usage of Docker as a whole, see the call out in https://opensearch.org/docs/latest/install-and-configure/install-opensearch/docker/#install-docker-and-docker-compose.